### DISCUSSION TITLE
LDF bug in Platformio that could affect AsyncWebServer

### DISCUSSION THREAD

**Discussion by vortigont** [mathieucarbou/ESPAsyncWebServer#51] [General]
Just for you information, I found an issue that might be a bug in Platformio.
Here is a [bug](https://github.com/platformio/platformio-core/issues/4940) ticket and a [PR](https://github.com/platformio/platformio-core/pull/4941) with fix.
And [here](https://community.platformio.org/t/how-to-properly-include-libraries-in-custom-libraries/39480/25) is some old discussion related for same in AsyncServer.
Mind if you may stumble upon it too.

**Comment by mathieucarbou** [comment]
thanks! didn't know.
